static/css/styles.css

```diff
@@ -6,6 +6,7 @@ body {
   width: 50%;
   text-align: center;
   padding-top: 5em;
+  background-image: url("paper.gif");
```

Comment:
Can you add this file to the repository? I am currently not able to view the image.